apps/src/code-studio/pd/professional_learning_landing/EnrolledWorkshops.jsx

```diff
@@ -78,6 +78,7 @@ class EnrolledWorkshopsTable extends React.Component {
           <Button
             onClick={() => this.openCertificate(workshop)}
             style={styles.button}
+            disabled={!workshop.attended}
```

Comment:
:+1: The way you've serialized out the relevant data makes this line, where we eventually use it to make a decision, _so nice._  Good choice.